src/processor/evict_seat.rs

```diff
@@ -98,8 +104,25 @@ pub fn process_evict_seat(program_id: &Pubkey, accounts: &[AccountInfo]) -> Prog
     // Retrieve seat manager seeds and check if signer is authorized
     let is_fully_authorized = *signer.key == seat_manager.load()?.authority;
 
+    // If we are doing eviction with an authorized delegate, we need to validate the signer
+    let mut is_authorized_delegate = false;
```

Comment:
Can we simplify by having a single boolean for `is_fully_authorized`? Don't feel strongly

Reply:
It would certainly be cleaner under the current design. I guess I writing for a possible future where there is additional granularity in permissions. I'll change it.